src/main/java/org/commcare/formplayer/util/StringUtils.java

```diff
@@ -9,6 +9,9 @@ public static String getFullUsername(String user, String domain, String host) {
     }
 
     public static String getFullUsername(String user, String domain) {
+        if (user.contains("@")) {
```

Comment:
since this method is only overloading the method before this, this change should actually be made in the previous method.

Reply:
You mean in MenuSession.getNextScreen correct?

Reply:
no there is another `getFullUsername` method that this method calls.